tests/unit/test_api.py

```diff
@@ -101,17 +101,11 @@ def test_provider_request_release_files(mock_old_api):
     [
         ([], False, []),
         (["allowed"], False, ["r1"]),
-        (
-            [],
-            True,
-            [
-                "r1",
-                "r2",
-                "r3",
-            ],
-        ),
+        # output checkers can't create requests unless explit permission for workspace
```

Comment:
I don't understand what creating requests has to do with getting requests

Reply:
We create a request on demand if one doesn't exist. For that one case, we pass create=True.